README.md

```diff
@@ -128,7 +128,6 @@ any form of iterator, different iterators have to be handled differently.
 | reduce                      | ☑    | ☑      | ☑             | ☑  |
 | sorted                      | ☐    | ☐      | ☑             | ☐  |
 | unique                      | ☐    | ☐      | ☑             | ☑  |
-| average                     | ☐    | ☐      | ☐             | ☑  |
```

Comment:
The argument for removing this doesn't hold much value. It exists as a method on C# enumerables. This table isn't focused on python or python naming schemes, selectively removing one C# method due to naming disputes undermines our ability to understand the prior art. Please revert this change. Otherwise looks ok to me.

Reply:
Amended.